### PR TITLE
ast/term.go: copy() nitpicks

### PR DESCRIPTION
**Pull request by srenatus**
It's nitpicking, really. Please play close attention when reviewing, though 😅

### CODE REVIEW

File: ast/term.go [hunk 1]
```diff
@@ -919,9 +916,8 @@ func (ref Ref) Extend(other Ref) Ref {
 	head.Value = String(head.Value.(Var))
 	offset := len(ref)
 	dst[offset] = head
-	for i := range other[1:] {
-		dst[offset+i+1] = other[i+1]
-	}
+
+	copy(dst[offset+1:], other[1:])
```
Comment (srenatus) [inline ast/term.go:920]: Here's my playground for fiddling with this... https://go.dev/play/p/oIwC0hpfau1 😄

Comment (philipaconrad) [reply ast/term.go:920]: The switch to [`copy`](https://pkg.go.dev/builtin#copy) makes the intent a bit more explicit and concise, which I like. :smile:

**Comment by philipaconrad** [comment]
@srenatus Thank you for your patience! I carefully inspecting the indexing, and it all looks good. Merging shortly. :+1:

**Comment by srenatus** [comment]
🙌 Thanks for taking the time to look at this closely. 🔍

**Pull request merged**